modules/vba_documents/app/workers/vba_documents/upload_processor.rb

```diff
@@ -97,6 +99,13 @@ def original_file_metadata(tempfile)
       }
     end
 
+    def validate_payload_size(tempfile)
+      unless tempfile.size.positive?
```

Comment:
I wanted to use `tempfile.size > 0` since it seems clearer to me given the context, but Rubocop insisted on using `positive?`.

Reply:
Oh man, this shouldn't grind my gears so much, but it does. I agree with your readability sensibilities.